ncclient/xml_.py

```diff
@@ -90,7 +87,7 @@ def register_namespace(prefix, uri):
 def to_xml(ele, encoding="UTF-8", pretty_print=False):
     "Convert and return the XML for an *ele* (:class:`~xml.etree.ElementTree.Element`) with specified *encoding*."
     xml = etree.tostring(ele, encoding=encoding, pretty_print=pretty_print)
-    if sys.version_info < (3,):
+    if sys.version < '3':
         return xml if xml.startswith('<?xml') else '<?xml version="1.0" encoding="%s"?>%s' % (encoding, xml)
     else:
         return xml.decode('UTF-8') if xml.startswith(b'<?xml') \
```

Comment:
Why we are using decode in these 2 lines. What will be the impact if we remove the decode. Can you please analyze?